src/android/AudioPlayer.java

```diff
@@ -757,4 +758,20 @@ public float getCurrentAmplitude() {
         }
         return 0;
     }
+
+    public void setRate(float speed) {
+        // Check for API 23+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
If `setRate` requires API 23, then we should add that as a quirk in the documentation stating that devices with a lower API level will be a no operation.